src/lib/components/ui/radio-button/radio-list.svelte

```diff
@@ -0,0 +1,75 @@
+<script lang="ts">
```

Comment:
@agazso Is `RadioControl` better name than `RadioList`?

Reply:
This is usually called `RadioGroup`.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio

Reply:
Sounds good, I'll rename